### ISSUE TITLE
Error on latest `devpi-server` and `devpi-postgresql`

### ISSUE THREAD

**Issue by qubidt** [devpi/devpi#884]
log and traceback:
```
.venv % devpi-server --role replica -c devpi-replica.yml --ldap-config devpi-ldap.yml
2022-04-06 14:39:03,670 INFO  NOCTX Loading node info from /home/bao/.devpi/server/.nodeinfo
2022-04-06 14:39:03,671 INFO  NOCTX wrote nodeinfo to: /home/bao/.devpi/server/.nodeinfo
2022-04-06 14:39:03,831 INFO  NOCTX Using 5 file download threads.
2022-04-06 14:39:03,918 INFO  NOCTX running with role 'replica'
2022-04-06 14:39:03,922 INFO  NOCTX Using secret file 'devpi-server.secret'.
2022-04-06 14:39:05,370 INFO  NOCTX Found plugin devpi-postgresql-3.0.0.
2022-04-06 14:39:05,370 INFO  NOCTX Found plugin devpi-ldap-2.1.0.
2022-04-06 14:39:05,371 INFO  NOCTX Found plugin devpi-web-4.0.8.
2022-04-06 14:39:05,371 INFO  NOCTX Found plugin devpi-semantic-ui-0.2.2.
2022-04-06 14:39:05,482 INFO  NOCTX Using /home/bao/.devpi/server/.indices for Whoosh index files.
2022-04-06 14:39:05,503 INFO  [ASYN] Starting asyncio event loop
2022-04-06 14:39:05,504 INFO  [FREPQ] Queuing files for possible download from master
2022-04-06 14:39:05,510 INFO  NOCTX devpi-server version: 6.5.0
2022-04-06 14:39:05,510 INFO  NOCTX serverdir: /home/bao/.devpi/server
2022-04-06 14:39:05,510 INFO  NOCTX uuid: ba9fe4519c854988a2be410e07d13c22
2022-04-06 14:39:05,510 INFO  NOCTX serving at url: http://0.0.0.0:58008 (might be http://[0.0.0.0]:58008 for IPv6)
2022-04-06 14:39:05,510 INFO  NOCTX using 50 threads
2022-04-06 14:39:05,510 INFO  NOCTX bug tracker: https://github.com/devpi/devpi/issues
2022-04-06 14:39:05,510 INFO  NOCTX Hit Ctrl-C to quit.
2022-04-06 14:39:05,514 INFO  Serving on http://0.0.0.0:58008
2022-04-06 14:39:05,600 INFO  [REP] fetching http://10.10.10.10:5055/+changelog/1443-?initial_fetch
2022-04-06 14:39:08,941 INFO  [REP] fetching http://10.10.10.10:5055/+changelog/1443-
2022-04-06 14:39:09,018 ERROR [FREPQ] Exception in thread 'InitialQueueThread'
Traceback (most recent call last):
  File ".venv/lib/python3.10/site-packages/devpi_server/mythread.py", line 40, in run
    result = threading.Thread.run(self)
  File "/usr/lib/python3.10/threading.py", line 946, in run
    self._target(*self._args, **self._kwargs)
  File ".venv/lib/python3.10/site-packages/devpi_server/replica.py", line 1082, in thread_run
    for item in relpaths:
  File ".venv/lib/python3.10/site-packages/devpi_postgresql/main.py", line 275, in iter_relpaths_at
    changes = self.get_changes(serial, absent)
TypeError: Connection.get_changes() takes 2 positional arguments but 3 were given
2022-04-06 14:39:09,019 INFO  [ASYN] The asyncio event loop stopped
2022-04-06 14:39:09,019 INFO  [ASYN] Thread 'AsyncioLoopThread' ended
Traceback (most recent call last):
  File ".venv/bin/devpi-server", line 8, in <module>
    sys.exit(main())
  File ".venv/lib/python3.10/site-packages/devpi_server/main.py", line 73, in main
    return _main(pluginmanager, argv=argv)
  File ".venv/lib/python3.10/site-packages/devpi_server/main.py", line 128, in _main
    return xom.main()
  File ".venv/lib/python3.10/site-packages/devpi_server/main.py", line 309, in main
    return xom.thread_pool.run(wsgi_run, xom, app)
  File "/usr/lib/python3.10/contextlib.py", line 281, in helper
    return _GeneratorContextManager(func, args, kwds)
  File "/usr/lib/python3.10/contextlib.py", line 103, in __init__
    self.gen = func(*args, **kwds)
  File ".venv/lib/python3.10/site-packages/devpi_server/mythread.py", line 103, in run
    raise self._fatal_exc
  File ".venv/lib/python3.10/site-packages/devpi_server/mythread.py", line 40, in run
    result = threading.Thread.run(self)
  File "/usr/lib/python3.10/threading.py", line 946, in run
    self._target(*self._args, **self._kwargs)
  File ".venv/lib/python3.10/site-packages/devpi_server/replica.py", line 1082, in thread_run
    for item in relpaths:
  File ".venv/lib/python3.10/site-packages/devpi_postgresql/main.py", line 275, in iter_relpaths_at
    changes = self.get_changes(serial, absent)
TypeError: Connection.get_changes() takes 2 positional arguments but 3 were given
```

devpi versions:
```
.venv $ pip list | grep devpi
devpi-common         3.6.0
devpi-ldap           2.1.0
devpi-postgresql     3.0.0
devpi-semantic-ui    0.2.2
devpi-server         6.5.0
devpi-web            4.0.8
```

Taking a cursory look at the code, this seems to be pretty straightforward (unless there's some runtime shenanigans going on I'm not seeing). Seems like this call to `get_changes()`:

https://github.com/devpi/devpi/blob/fce31fd42f331cfcc535750ed581b57729e1fa8b/postgresql/devpi_postgresql/main.py#L262-L275

doesn't match the parameters for the function definition:

https://github.com/devpi/devpi/blob/fce31fd42f331cfcc535750ed581b57729e1fa8b/postgresql/devpi_postgresql/main.py#L441-L451

**Comment by fschulze** [comment]
Thanks for the report. It looks like the tests are missing coverage for this part.

**Comment by fschulze** [comment]
Fix released with 3.0.1.
